packages/nimble-components/src/table/tests/table-column-sizing.spec.ts

```diff
@@ -441,6 +495,12 @@ describe('Table Interactive Column Sizing', () => {
                 `${columnSizeTest.name}`,
                 // eslint-disable-next-line @typescript-eslint/no-loop-func
                 async () => {
+                    element.columns.forEach((column, i) => {
```

Comment:
It might be worth writing a test (or potentially updating these tests) to verify that each column has the expected fractional width after the resize operation. I could see that potentially being a pain because of the result being a seemingly arbitrary value with a lot of decimal places, but I think it would be beneficial to have at least one unit test that verifies that the "normalization" logic is working as expected.

Reply:
We agreed that [this test](https://github.com/ni/nimble/blob/ab995f09749e33f9fdcb5a5975a8c1d84d8aca00/packages/nimble-components/src/table/tests/table-column-sizing.spec.ts#L492) adequately covers this concern along with the other new tests.